CMakeLists.txt

```diff
@@ -60,22 +63,7 @@ ELSE (WIN32)
 
   SET(CMAKE_C_FLAGS_DEBUG "-DDEBUG -g")
   SET(CMAKE_C_FLAGS_RELEASE "-DNDEBUG -O2 -Wuninitialized")
-ENDIF (WIN32)
+ENDIF (MSVC)
 
 
 ADD_SUBDIRECTORY(src)
-ADD_SUBDIRECTORY(test)
```

Comment:
It will be easier to add an option:
```cmake
option(YAIL_BUILD_TOOLS "Build tools" ON)

# ...

if(YAJL_BUILD_TOOLS)
  add_subdirectory(...)
endif()
```

Also, that's something that you can send upstream, so we don't have to maintain this fork.

Reply:
With this approach for the sake of minimal change, we can give up formatting rules:
```cmake
if(YAJL_BUILD_TOOLS)

# keep original indentation and style
ADD_SUBDIRECTORY(reformatter)
ADD_SUBDIRECTORY(verify)
#...
```

Reply:
Unfortunately the upstream repo seems almost abandoned and has not accepted PRs in many years. I think forking is our only option here.

Reply:
Would you like me to create multiple options? I suppose that besides `YAJL_BUILD_TOOLS` you would like to have `YAJL_BUILD_EXAMPLE` and `YAJL_BUILD_TEST` as well?

Reply:
Hm, Okay, I see.

Reply:
> Would you like me to create multiple options? I suppose that besides YAJL_BUILD_TOOLS you would like to have YAJL_BUILD_EXAMPLE and YAJL_BUILD_TEST as well?

Up to you.